File: src/API/src/Modules/Clients/Clients.Application/ViewModels/ClientFilterLookupViewModel.cs
namespace Clients.Application.ViewModels
{
    public class ClientFilterLookupViewModel
    {
        public List<LookupViewModel> Countries { get; set; } = new();
        public List<LookupViewModel> Industries { get; set; } = new();
        public List<LookupViewModel> Statuses { get; set; } = new();
        public List<ClientUserLookupViewModel> AccountOwners { get; set; } = new();
    }
}

File: src/ClientApp/src/app/pages/clients/view-models/client-filter-lookup.view-model.ts
import { LookupViewModel } from '../../leads/view-models/lookup.view-model';
import { ClientUserLookupViewModel } from './client-user-lookup.view-model';

export interface ClientFilterLookupViewModel {
    countries: LookupViewModel[];
    industries: LookupViewModel[];
    statuses: LookupViewModel[];
    accountOwners: ClientUserLookupViewModel[];
}
